src/corehost/cli/dll/CMakeLists.txt

```diff
@@ -32,6 +40,8 @@ else()
     list(APPEND SOURCES ../../common/pal.unix.cpp)
 endif()
 
+add_definitions(-D_NO_ASYNCRTIMP)
+add_definitions(-D_NO_PPLXIMP)
 add_definitions(-DCOREHOST_MAKE_DLL=1)
 
 add_library(hostpolicy SHARED ${SOURCES})
```

Comment:
Given that the host will be called dotnet.exe, we should rename this to be "dotnethostpolicy.dll" and "hostfxr.dll" to "dotnethostfxr.dll"